### PR TITLE
Fix error throwing when `supports-selector-fn` has pseudo-classes or pseudo-elements as argument.

### PR DESCRIPTION
**Pull request by Semigradsky**
Related to #1391

### CODE REVIEW

**Comment by Semigradsky** [comment]
@dvoytenko @cpmsmith Could you please try this change?

File: test/supports.test.js [hunk 1]
```diff
@@ -155,4 +155,12 @@ describe('process()', () => {
     supports.process(rule)
     expect(rule.params).toEqual('(color black) and not ((-moz-a: 1) or (a: 1))')
   })
+
+  it("shouldn't throw errors", () => {
+    let rule = { params: ' (selector( :nth-child(1n of a, b) )) or (c: b(1)) ' }
```
Comment (ai) [inline test/supports.test.js:160]: Can you add also a case without white-space between `selector(` and `:`?

Comment (Semigradsky) [reply test/supports.test.js:166]: I have added one more test.

**Comment by ai** [comment]
Released in 10.2.5

**Pull request merged**